pyre/check.py

```
@@ -2424,6 +2442,12 @@ def _retry_performance_gate(
             pyre_times.append(elapsed)
         return statistics.median(pyre_times), statistics.median(baseline_times)
 
+    def _startup_drift(self, key):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_startup_drift`

(ANN202)